homanp/langchain-ui-api
# feat: add .env.example

## Summary

This PR adds the missing `.env.example` file to the repository, as mentioned in the GitHub issue: ".env.example file missing in repository". The file contains a sample environment variable needed for the project setup.

Fixes #5 

Depends on: None

## Test plan

1. Clone the repository.
2. Check for the presence of the `.env.example` file in the root directory.
3. Copy the `.env.example` file to `.env`.
4. Replace the placeholder with your actual API key or other required values.
5. Run the project and confirm that it works with the new `.env` file.

## Screenshots

N/A
